mh/pat.py

```diff
@@ -2357,6 +2363,28 @@ def sendAnsLayerMediationUnlock(self, index, mediation_data, seq):
         data = struct.pack(">B", success)
         self.send_packet(PatID4.AnsLayerMediationUnlock, data, seq)
 
+    def sendNtcLayerHost(self, new_leader, seq):
+        """NtcLayerHost packet.
+        ID: 64411000
+        JP: レイヤのホスト通知
+        TR: Layer host notification
+        """
+        data = struct.pack(">HII", 0x08, 0x00, 0x00)  # unk short array patitem
+        data += pati.lp2_string(new_leader.capcom_id)
+        data += pati.lp2_string(new_leader.hunter_name)
+        self.server.layer_broadcast(self.session, PatID4.NtcLayerHost, data,
+                                    seq)
+
+    def leave_layer(self):
```

Comment:
‘leave_layer’ is not a proper name for what this function actually does, maybe a better name could be ‘notify_layer_departure’?